parity/main.rs

```diff
@@ -63,8 +64,8 @@ Options:
   -d --db-path PATH        Specify the database & configuration directory path [default: $HOME/.parity]
 
   --no-bootstrap           Don't bother trying to connect to any nodes initially.
-  --listen-address URL     Specify the IP/port on which to listen for peers [default: 0.0.0.0:30304].
-  --public-address URL     Specify the IP/port on which peers may connect [default: 0.0.0.0:30304].
+  --listen-address URL     Specify the IP/port on which to listen for peers.
+  --public-address URL     Specify the IP/port on which peers may connect.
```

Comment:
Defaults are now determined by the networking layer and not by docopts. 
"0.0.0.0" public address does not make sense.